CONTRIBUTING.md

```diff
@@ -26,31 +26,29 @@ Use IntelliJ IDEA Community Edition/Ultimate.
 
 3. (optional) If working on IntelliJ Ultimate, enable JavaScript and TypeScript plugin (for UI tests).
 
-4. (optional) Install the following non-bundled plugins from Marketplace:
+4. Install the following non-bundled plugins from Marketplace:
+    * [Kotlin plugin](https://plugins.jetbrains.com/plugin/6954-kotlin) will be useful for editing certain parts of UI, esp. dialogs.
+    * [Scala plugin](https://plugins.jetbrains.com/plugin/1347-scala) might be useful for editing UI tests.
+
+5. (optional) Install further non-bundled plugins from Marketplace:
    * [Grammar-Kit IntelliJ plugin](https://plugins.jetbrains.com/plugin/6606-grammar-kit) can be used instead of Gradle plugin
      to manually generate grammar and lexer code from `.bnf` and `.flex` files.
    * [HOCON plugin](https://plugins.jetbrains.com/plugin/10481-hocon) for `.conf` file support in UI tests
-   * [Kotlin plugin](https://plugins.jetbrains.com/plugin/6954-kotlin) will be useful for editing certain parts of UI, esp. dialogs.
    * [PsiViewer IntelliJ plugin](https://plugins.jetbrains.com/plugin/227-psiviewer) can be helpful to see parsing result on the `machete` file
      when running IntelliJ instance with the Git Machete plugin loaded.
-   * [Scala plugin](https://plugins.jetbrains.com/plugin/1347-scala) might be useful for editing UI tests.
 
-5. Enable annotation processing (for Lombok):
+6. Enable annotation processing (for Lombok):
    `File > Settings > Build, Execution, Deployment > Compiler > Annotation Processors > Enable Annotation Processing`.
```

Comment:
Ok, I see the same `File > Settings > ...` thing here and probably it might appear in other places in CONTRIBUTING.md file. Looks like we might need a separate issue for that case, so that descriptions will be valid both for Linux and macOs. Just a non-blocking comment from my side.

Reply:
Some fixed on this PR; opened https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1207 for a systematic walkthrough

Reply:
Actually, what it this path for Annotation Processing on macOS?

Reply:
<img width="889" alt="image" src="https://user-images.githubusercontent.com/49597713/193776799-c9ac7fbb-2b2b-44c1-a257-0a55b629453d.png">

Reply:
Not directly, but under the `IntelliJ IDEA` top-level menu category. On macOs there is always a top-level menu category named the same as the application itself, and there are Preferences...

Reply:
Awww okay